src/components/ValidationStatus.vue

```diff
@@ -1,113 +1,77 @@
-    </div>
-
-    <div class="status-bar-container d-flex align-items-center justify-content-end">
-      <b-form-checkbox
-        data-test="validation-toggle"
-        class="h-100 d-flex align-items-center"
-        v-model="autoValidate"
-        switch
-      >
-        {{ $t('Auto validate') }}
-      </b-form-checkbox>
-
-      <div class="divider"/>
-
-      <div data-test="validation-list-toggle" class="status-bar-container__status" @click="toggleValidationPanel = !toggleValidationPanel; toggleWarningsPanel = false">
-        <div class="d-flex align-items-center">
-          <span class="status-bar-container__status-text">{{ $t('Problems') }} {{ numberOfValidationErrors }}</span>
-          <font-awesome-icon class="status-bar-container__status-icon h-100" :style="{ color: statusColor }" :icon="statusIcon" />
-          <font-awesome-icon class="status-bar-container__status-ellipsis" :icon="ellipsisIcon" />
+      {{ $t('Auto validate') }}
+    </b-form-checkbox>
+
+    <span class="divider" />
+
+    <div v-if="isProblemsPanelDisplayed && numberOfProblems" class="validation-container position-absolute text-left" data-test="validation-list">
+      <div class="validation-container-list d-flex align-items-baseline" v-for="error in errorList" :key="`${error.id}_${error.errorKey}`">
+        <font-awesome-icon
+          class="status-bar-container-status-icon ml-1 mr-2 mt-1"
+          :style="{ color: isErrorCategory(error) ? errorColor : warningColor }"
+          :icon="isErrorCategory(error) ? faTimesCircle: faExclamationTriangle"
+        />
+        <div class="validation-container-list-message">
+          <h6 class="text-capitalize mb-0">{{ error.errorKey }}</h6>
+          <p class="mb-0"><em>{{ error.message }}.</em></p>
+          <p class="mb-0" v-if="error.id"><strong>{{ $t('Node ID') }}:</strong> {{ error.id }}</p>
         </div>
       </div>
 
-      <div class="divider"/>
-
-      <div class="status-bar-container__status" @click="toggleWarningsPanel = !toggleWarningsPanel; toggleValidationPanel = false">
-        <div class="d-flex align-items-center">
-          <span class="status-bar-container__status-text">{{ $t('Warnings') }} {{ warnings.length }}</span>
-          <font-awesome-icon
-            class="status-bar-container__status-icon h-100"
-            :style="{ color: warnings.length > 0 ? warningColor : validColor }"
-            :icon="warnings.length > 0 ? faExclamationTriangle : faCheckCircle"
-          />
-          <font-awesome-icon class="status-bar-container__status-ellipsis" :icon="ellipsisIcon" />
+      <div class="validation-container-list d-flex align-items-baseline" v-for="(warning, index) in warnings" :key="index">
+        <font-awesome-icon class="status-bar-container-status-icon ml-1 mr-2 mt-1" :style="{ color: warningColor }" :icon="faExclamationTriangle" />
+        <div class="validation-container-list-message">
+          <h6 class="text-capitalize mb-0">{{ warning.title }}</h6>
```

Comment:
Is the `h6` here only used for the styling? I actually don't see any other heading levels used on the page, so skipping to level 6 seems like it's done for styling as oppose to creating some sort of semantic hierarchy/outline.

See https://developers.google.com/web/fundamentals/accessibility/how-to-review?utm_source=lighthouse&utm_medium=devtools#take_advantage_of_headings_and_landmarks:
> Make proper use of h1-h6 hierarchy. Think of headings as tools to create an outline for your page. Don’t rely on the built-in styling of headings; instead, consider all headings as if they were the same size and use the semantically appropriate level for primary, secondary, and tertiary content. Then use CSS to make the headings match your design.

You may not even need heading elements here, as there's no real sections that need a title. I think using a description/definition list here might work well: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl.

Reply:
I did think about a definition list, let me try that out.